changelog/2801.breaking.rst

```diff
@@ -0,0 +1 @@
+Helioviewer client now uses Helioviewer API version 2 exclusively.
```

Comment:
Given the API changes introduced in this PR, we should massively expand this changelog entry.